app/assets/javascripts/reducers/users.js

```diff
@@ -23,12 +24,32 @@ const SORT_DESCENDING = {
 
 export default function users(state = initialState, action) {
   switch (action.type) {
-    case RECEIVE_USERS: return {
+    case RECEIVE_USERS: {
+      // Get the sorting key from if available from Redux store or else use last_name
+      let sort_key = state.sort.key || 'last_name';
```

Comment:
setting it to `last_name` gets overridden by the assignment at L41 - what's it for?

Reply:
It does , but in case `if` (L35) statements is true which is if even one user have `real_name`  then it wont be overridden. As for the `last_name` it is used if  any user have real_name or else sorting will fall back to `username`.